README.md

```diff
@@ -8,9 +8,13 @@
 </p>
 
 <p align="center">
+[📖 Documentation][docs]
+
 Chain-bench is an open-source tool for auditing your software supply chain stack for security compliance based on a new 
   <a href="docs/CIS-Software-Supply-Chain-Security-Guide-v1.0.pdf">CIS Software Supply Chain benchmark</a>.
 The auditing focuses on the entire SDLC process, where it can reveal risks from code time into deploy time. To win the race against hackers and protect your sensitive data and customer trust, you need to ensure your code is compliant with your organization’s policies.
+
+Read more in the [Chain-bench Documentation][docs]
```

Comment:
can we mark this out until we finish with the mk doc work?

Reply:
No because that'd make this PR just remove docs from the README and hide them in the docs dir where people would have to actively go find them

With all due respect this would never have been an issue if I wasn't asked to migrate my docs changes from the readme to the docs dir. I'm afraid I'm not going to put more effort in to put it all back

https://github.com/aquasecurity/chain-bench/pull/13#issuecomment-1157431880

https://github.com/aquasecurity/chain-bench/pull/13#issuecomment-1157453407